### ISSUE TITLE
Fast search bug

### ISSUE THREAD

**Issue by guilhermecugler** [Akascape/CTkScrollableDropdown#30]
If you type too quickly in the autocomplete live search, the letters appear in the wrong order and take a long time to appear.

**Comment by AnonymousVibrate** [comment]
Yeah, I also experience something like this in the latest version. Still no fix to this ?
